doc/instances.md

```diff
@@ -321,6 +321,7 @@ maas.subnet.ipv6         | string  | -                 | no       | yes     | MA
 boot.priority            | integer | -                 | no       | no      | Boot priority for VMs (higher boots first)
 vlan                     | integer | -                 | no       | no      | The VLAN ID to use for untagged traffic (Can be `none` to remove port from default VLAN)
 vlan.tagged              | integer | -                 | no       | no      | Comma delimited list of VLAN IDs to join for tagged traffic
+security.port\_isolation | boolean | false             | no       | no      | Prevent the instance from allowing ports to communicate directly with each other (enables port\_isolation)
```

Comment:
From my understanding of the bridge `isolated` feature (https://man7.org/linux/man-pages/man8/bridge.8.html) it only prevents communication with other isolated ports.

Whereas the explanation here suggests that enabling this feature on one instance NIC would prevent it communicating with other instances, which isn't true unless they too enable the feature.

Suggest tweaking this to say "Prevent the NIC from communicating with other NICs in the network that have port isolation enabled"